settings.gradle

```diff
@@ -32,10 +32,23 @@ pluginManagement {
     }
   }
   repositories {
-    gradlePluginPortal()
     mavenCentral()
     google()
-    jcenter()
+    gradlePluginPortal()
```

Comment:
Not sure if this changed, but at least initially this also served jcenter artifacts so you might want `exclusiveContent` here as well.

Reply:
removed because I don't think we need anything from jcenter here